src/dotnet/commands/dotnet-new3/HelpFormatter.cs

```diff
@@ -0,0 +1,301 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.DotNet.Tools.New3
+{
+    public class HelpFormatter
```

Comment:
This looks like generic functionality that's not specific to the new3 command. We probably want this `internal` and if it's something that's useful elsewhere, we might want to locate it in Utils.

Do you have unit tests for this?